### ISSUE TITLE
New Features Request

### ISSUE THREAD

**Issue by zero77** [WhisperSystems/Signal-Android#5378]
Can you please add these new features.

Have the option to add a status message next to your name in the contacts list.

Have the ability to add a profile picture.

Have the ability to send a message picture video et cetera to multiple contacts at once.

Add more notification options like turn the screen on if the phone is locked when you get a new message. Also more vibration options like vibration strength pattern and more LED options like pattern and frequency.

Can you also add a message on the call screen to alert you that the call has been muted or is on hold and who it was by like you or the other contact.

**Comment by agrajaghh** [comment]
Why did you delete the issue template? Did you even read it? You're wasting the time of everybody following the repo (and the main devs!) by not searching existing issues first...

> Have the option to add a status message next to your name in the contacts list.

#2197 

> Have the ability to add a profile picture.

#706

> Have the ability to send a message picture video et cetera to multiple contacts at once.

#3301

> Add more notification options like turn the screen on if the phone is locked when you get a new message.

that sounds like something the OS should do...

> Also more vibration options like vibration strength pattern and more LED options like pattern and frequency.

there is a LED pattern option! Maybe check the actual options first?
vibrate: #230

> Can you also add a message on the call screen to alert you that the call has been muted or is on hold and who it was by like you or the other contact.

there are already missed call notifications!

**Comment by zero77** [comment]
Yes I did read the template and I did search but obviously not well enough sorry.

> that sounds like something the OS should do...
Yes that's what I thought but it doesn't seem to be available only when sending SMS from the default app and I am using an LG G3 running android 6.0.

> there are already missed call notifications!

That’s not what I meant. I meant having a permanent message that notifies the user the call has been muted or is on hold and tells them who put the call on mute or on hold.
